stanford/roles/deployment/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+DEPLOYMENT_SSL_CERTIFICATE_ID: 'arn:aws:acm:<REGION>:<ACCOUNT>:certificate/<GUID>'
```

Comment:
Eventually, it will be nice to perform a run-time lookup of this, based on tags.